tools/js2asar.py

```diff
@@ -27,7 +27,11 @@ def copy_files(source_files, output_dir, folder_name):
     # Files that aren't in the default_app folder need to be put inside
     # the temp one we are making so they end up in the ASAR
     if not os.path.normpath(source_file).startswith(folder_name + os.sep):
-      output_path = os.path.join(output_dir, folder_name, source_file)
+      inner = os.sep + folder_name + os.sep
+      if inner in source_file:
```

Comment:
yikes, what is this even doing and why?

Reply:
Dealt with this entire fire of a file in #16718